src/sentry/conf/server.py

```diff
@@ -2512,5 +2512,14 @@ def build_cdc_postgres_init_db_volume(settings):
 SENTRY_ISSUE_ALERT_HISTORY = "sentry.rules.history.backends.postgres.PostgresRuleHistoryBackend"
 SENTRY_ISSUE_ALERT_HISTORY_OPTIONS = {}
 
+SUPERUSER_ACCESS_CATEGORIES = [
```

Comment:
didn't we want to make this private? (i.e., not in the `sentry` repo)

Reply:
@maxiuyuan why did you resolve this conversation?

Reply:
oops, i removed this on my working branch and not this one, will remove this too